shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/database/type/DatabaseTypeFactory.java

```diff
@@ -52,4 +59,41 @@ public static DatabaseType getInstance(final String name) {
     public static Collection<DatabaseType> getInstances() {
         return ShardingSphereServiceLoader.getServiceInstances(DatabaseType.class);
     }
+    
```

Comment:
Why change main codes if the PR is add test cases only?

Reply:
In issues,  add more unit test for DatabaseTypeFactory.
- DatabaseTypeFactory#getDatabaseType
But I didn't find this method, so I'll define it myself.

Reply:
I understand. I am learning the code and have finished modifying it